observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc

```diff
@@ -78,6 +63,13 @@ There are currently known issues:
 [id="distributed-tracing-rn_3-1_jaeger-release-notes"]
 === {JaegerName}
 
+The {JaegerName} is provided through the {JaegerOperator} Operator.
```

Comment:
Same issue with the attribute not rendering properly here https://73592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases#distributed-tracing-rn_3-1_jaeger-release-notes